Source/Svg.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29215.179
+# Visual Studio Version 17

```

Comment:
Not sure if we shall change the version - it means it won't be opened with versions < 2022.

Reply:
I'll revert that